config/initializers/assets.rb

```diff
@@ -26,6 +26,4 @@
 Rails.application.config.assets.precompile += %w( potlatch2/FontLibrary.swf )
 Rails.application.config.assets.precompile += %w( potlatch2/locales/*.swf )
 Rails.application.config.assets.precompile += %w( help/introduction.* )
-Rails.application.config.assets.precompile += %w( iD/img/*.svg iD/img/*.png iD/img/*.gif )
-Rails.application.config.assets.precompile += %w( iD/img/pattern/*.png )
-Rails.application.config.assets.precompile += %w( iD/locales/*.json )
```

Comment:
Thanks @tomhughes this looks great!  :shipit: 

http://tomh.apis.dev.openstreetmap.org/edit#map=17.08/40.71285/-74.00614

<img width="767" alt="screenshot 2016-03-03 22 18 19" src="https://cloud.githubusercontent.com/assets/38784/13517082/2c1b6f7e-e18e-11e5-947b-0e19044dee2f.png">